### DISCUSSION TITLE
Do you guys have support for domain change from <appname>.reflex.run? (to name.tld) (This is self-hosting but a domain change for *.reflex.run)

### DISCUSSION THREAD

**Discussion by itsmeadarsh2008** [reflex-dev/reflex#3398] [General]
Do you guys have support for domain change from <appname>.reflex.run to name.tld? This is for self-hosting, and I would like to change the domain for all subdomains under reflex.run.

**Comment by picklelo** [comment]
We don't have support for custom domains right now, but I'd recommend looking into a redirect service like https://redirect.pizza - I've used this to have custom domains on sites hosted on the reflex hosting service.

**Reply by picklelo** [reply]
If you're self hosting outside of our hosting service, you should be able to use custom domains as normal. The above only applies for when you're using Reflex hosting service.
